src/AccordionPanelGroup.jsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import cx from 'classnames';
+
+export const ACCORDIONPANELGROUP_CLASS = 'accordionPanelGroup';
+/**
+ * @module AccordionPanelGroup
+ */
+class AccordionPanelGroup extends React.Component {
+	constructor(props) {
+		super(props);
+		this.accordionPanels = this.props.accordionPanels.map((accordionPanel, i) => {
+			return this.clonePanels(accordionPanel, i, accordionPanel.props.isOpen);
```

Comment:
yeah, changing this to `clonePanel` makes sense. 

The `isOpen` argument is either provided by the props of the `AccordionPanel`, or provided by some logic in `AccordionPanelGroup`. After giving some refactoring a shot, I feel like keeping the argument makes it easier to understand when we're relying on the panel props vs. explicitly setting `isOpen`.